.drone/scripts/create_tag.sh

```diff
@@ -20,5 +20,5 @@ chmod 500 "/${HOME}/.ssh/"* -R
 version="$(cat .data.json | jq -r '.current_pkgver + "-" + .current_pkgrel')"
 
 # Create and push release
-git tag -f "v${version}-${release_type}" -am ""
+git tag -f "v${version}-${release_type}" -am "Bump debian version to v${version} [CI SKIP]"
```

Comment:
We shouldn't need to specify that we're tagging the Debian version, as the tag that gets published is also used in AUR and MPR packages.

Just a general note too, but the `[CI SKIP]` suffix shouldn't be needed on tags, as Drone hasn't ever run a pipeline when I made and pushed them.